themes/base16.ini

```diff
@@ -0,0 +1,82 @@
+[base]
+default          = none
+unknown-token    = 1,bold
+commandseparator = none
+redirection      = none
+here-string-tri  = 14
+here-string-text = bg:11
+here-string-var  = 1,bg:11
+exec-descriptor  = 9,bold
+comment          = 8
+correct-subtle   = 12
+incorrect-subtle = 1
+subtle-separator = 12
+subtle-bg        = bg:10
+secondary        =
```

Comment:
Is something missing here?

Reply:
A secondary theme doesn't make sense with base16 because any secondary theme you pick may be completely unreadable, especially if you're using a ligth terminal theme.

Reply:
In that case, why not comment it out like line below?

Reply:
Done.